front/src/modules/pathfinding/utils.ts

```diff
@@ -131,10 +128,16 @@ export const upsertViasInOPs = (ops: SuggestedOP[], pathSteps: PathStep[]): Sugg
  * Some OPs have same uic so we need to check also the ch (can be still not enough
  * probably because of imports problem).
  * If the vias has no uic, it has been added via map click and we know it has an id.
+ * @param withKP - If true, we check the kp compatibility instead of the name.
+ * It is used in the times and stops table to check if an operational point is a via.
  */
-export const isVia = (vias: PathStep[], op: SuggestedOP) =>
+export const isVia = (vias: PathStep[], op: SuggestedOP, withKP = false) =>
```

Comment:
Is there some cases where we prefer to check the name instead of the kp ? I think we can even always check the kp
```suggestion
export const isVia = (vias: PathStep[], op: SuggestedOP) =>
```

Reply:
I had to do that because of import issues. Sometimes, we want to check isVia for ops which have same uic/ch but different kps (so we pass true as third arg) and sometimes they have same uic/ch/kp so we need to compare them with their names..

Reply:
Ok, then can you just add a const so we don't have an argument to true in the code ? Like this for instance
```
const WITH_KP = true;
isVia(pathSteps, rowData, WITH_KP)
```

Reply:
Done